### ISSUE TITLE
Private Flag: show icon

### ISSUE THREAD

**Issue by Shrekmachine** [janvv/outlook-taskboard#20]
can we show an icon when the task is private

**Comment by janvv** [comment]
Changing the background color isn't enough? 
But yes, the halflings-lock icon could be added.
I think this only makes sense when you are not using the privacy filter.

**Comment by Shrekmachine** [comment]
Ah, yellow is "private". Icon appreciated anyway. just for those with issues for colors.
Indeed I switched of the Filter for separation of private tasks.
Even more I would prefer to change the behavior of the private switch to:
Private filter on: Show only public tasks
Private filter off: Show all tasks
does that make sense?

**Comment by janvv** [comment]
Yes, that makes sense. I am considering to make this more flexible by configuration.
In my case I am using the Kanban board for my work tasks and for my personal tasks at home. So the way it works currently perfectly fits in my situation.
